core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -199,9 +198,15 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
         .registerContributorDashboardViewSuggestionForReview('Translation');
       this.heading = 'Review Translation Contributions';
     }
+    const suggestionIdsList = Object.keys(this.suggestionIdToContribution);
```

Comment:
```suggestion
    const suggestionIds = Object.keys(this.suggestionIdToContribution);
```
Can drop "list" since it's redundant.

Reply:
Done